std/container/rbtree.d

```diff
@@ -1670,6 +1671,16 @@ assert(equal(rbt[], [5]));
     }
 
     /**
+    Returns a user-friendly string representation by writing to the $(D sink).
+    For more info, see $(D std.format.formatValue).
+    */
+    void toString(scope void delegate(const(char)[]) sink, FormatSpec!char fmt) const {
```

Comment:
This returns void, so starting the doc with "Returns" is not correct.

I think this might be acceptable:

```
Formats the RedBlackTree into a sink function. For more info see $(D std.format.formatValue)
```